python/ray/tune/examples/genetic_example.py

```diff
@@ -24,7 +24,7 @@ def michalewicz_function(config, reporter):
     y = np.dot(sin_x, sin_z)
 
     # Negate y since we want to minimize y value
-    reporter(timesteps_total=1, neg_mean_loss=-y)
+    reporter(timesteps_this_iter=1, neg_mean_loss=-y)
```

Comment:
this doesn't need to be changed right?

Reply:
Sorry, the old function runner seemed to have an odd behavior where timesteps_total was converted to timesteps_this_iter before being returned. I just went around and changed all timesteps_total using the function API as a precaution. Given this runs only for one iteration, this can be reverted.